doc/docs/modules/ROOT/pages/neo4j-cluster.adoc

```diff
@@ -0,0 +1,30 @@
+
+[#neo4j_causal_cluster]
+== Using with Neo4j Causal Cluster
```

Comment:
If we don't include the pages then we should start the pages with a document title (i.e. in this case `= Using with Neo4j Causal Cluster`)